app/views/govuk_publishing_components/components/docs/radio.yml

```diff
@@ -95,6 +109,42 @@ examples:
         text: "Yes"
       - value: "no"
         text: "No"
+  with_custom_heading_size:
```

Comment:
I think there's too many variations on the same thing and it's getting confusing. Suggestion:

- Keep `with_legend`
- Remove `with_heading`
- Rename `with_custom_heading_size` to `with_heading` and use it to demonstrate both the heading_level and heading_size options
- Remove `with_custom_heading_level`
- Create a new example `with_heading_1` that demonstrates the `if heading level is 1 and heading_size is not set` example

I think that covers all the options, but point out if I've missed any.